recipe/meta.yaml

```diff
@@ -42,6 +42,7 @@ outputs:
     requirements:
       run:
         - {{ pin_subpackage('importlib-metadata', max_pin="x.x.x") }}
+        - python
```

Comment:
```suggestion
```
This is not required, as being noarch generic.

Reply:
I think it was added by `anaconda-linter --fix`. It should make an exception for noarch generic